src/routers/SectionRouteRouter.js

```diff
@@ -0,0 +1,58 @@
+// @flow
+import type Request from 'express';
+import AnalyticsUtils from '../utils/AnalyticsUtils';
+import ApplicationRouter from '../appdev/ApplicationRouter';
+import LogUtils from '../utils/LogUtils';
+import SectionRouteUtils from '../utils/SectionRouteUtils';
+
+class SectionRouteRouter extends ApplicationRouter<Array<Object>> {
```

Comment:
i think we should just call this `RouteRouter`, and maybe put it in a `/v2` folder and separate the old stuff into a `/v1` folder